go.mod

```diff
@@ -101,4 +101,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20221109080512-912af81e4fa2
```

Comment:
just curious about the reasoning of this ? ...since no occurrence of the `SetApproved` functions are present.

Reply:
I just wanted to make sure I would upgrade to the latest version of toolchain-common and apply the needed changes. In this particular case, there is none, but this PR proves that there will be no breakage. 
But granted, maybe it was not needed 🤷‍♂️

Reply:
yes, see https://github.com/codeready-toolchain/host-operator/pull/702

Reply:
sorry, I've went only through the PR's linked in slack! 
thanks for the clarification, so that I know how to proceed in cases of changes to toolchain-common.